portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -60,9 +94,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   /**
    * Converts an ArrayList instance into a JSON string using the Gson library.
    */
-  private <T> String convertToJsonUsingGson(ArrayList<T> messages) {
+  private static final <T> String convertToJsonUsingGson(ArrayList<T> messages) {
+    Gson GSON = new Gson();
```

Comment:
Actually, I would suggest make it private static final.
Any reason to change it back?

Reply:
I think there's a conflict between this and last merge. I'll resolve this conflict.